it/repositories/sift/ApplicationSiftRepositorySpec.scala

```diff
@@ -0,0 +1,128 @@
+import model.Phase3TestProfileExamples.phase3TestWithResult
+import model.ProgressStatuses.PHASE3_TESTS_PASSED
+import model._
+import model.command.ApplicationForSift
+import model.persisted.{ PassmarkEvaluation, SchemeEvaluationResult }
+import org.scalatest.concurrent.ScalaFutures
+import repositories.onlinetesting.Phase2EvaluationMongoRepositorySpec.phase2TestWithResult
+import repositories.{ CollectionNames, CommonRepository }
+import testkit.{ MockitoSugar, MongoRepositorySpec }
+
+class ApplicationSiftRepositorySpec extends MongoRepositorySpec with ScalaFutures with CommonRepository with MockitoSugar {
+
+  val collectionName: String = CollectionNames.APPLICATION
+
+  val Commercial: SchemeId = SchemeId("Commercial")
+  val European: SchemeId = SchemeId("European")
+  val Sdip: SchemeId = SchemeId("Sdip")
+  val Generalist: SchemeId = SchemeId("Generalistt status")
+  val ProjectDelivery = SchemeId("Project Delivery")
+  val schemeDefinitions = List(Scheme(Commercial, "", "", requiresSift = true),
+    Scheme(ProjectDelivery, "", "", requiresSift = true),
+    Scheme(Generalist, "", "", requiresSift = true))
+
+  def repository: ApplicationSiftMongoRepository = applicationSiftRepository(schemeDefinitions)
+
+  "next Application for sift" should {
+    "ignore applications in incorrect statuses and return only the Phase3 Passed_Notified applications that are eligible for sift" in {
+      insertApplicationWithPhase3TestNotifiedResults("appId1", SchemeId("Commercial"), EvaluationResults.Green).futureValue
+
+      insertApplicationWithPhase3TestNotifiedResults("appId2", SchemeId("Commercial"), EvaluationResults.Green).futureValue
+      updateApplicationStatus("appId2", ApplicationStatus.PHASE3_TESTS_PASSED)
+
+      insertApplicationWithPhase3TestNotifiedResults("appId3", SchemeId("Commercial"), EvaluationResults.Green).futureValue
+      updateApplicationStatus("appId3", ApplicationStatus.PHASE3_TESTS_FAILED)
+
+      insertApplicationWithPhase3TestNotifiedResults("appId4", SchemeId("Project Delivery"), EvaluationResults.Green).futureValue
+
+      insertApplicationWithPhase3TestNotifiedResults("appId5", SchemeId("Finance"), EvaluationResults.Green).futureValue
+
+      insertApplicationWithPhase3TestNotifiedResults("appId6", SchemeId("Generalist"), EvaluationResults.Red).futureValue
+
+      val appsForSift = repository.nextApplicationsForSiftStage(10).futureValue
+      appsForSift mustBe List(
+        ApplicationForSift("appId1", PassmarkEvaluation("", Some(""),
+          List(SchemeEvaluationResult(SchemeId("Commercial"), EvaluationResults.Green.toPassmark)), "", Some(""))),
+        ApplicationForSift("appId4", PassmarkEvaluation("", Some(""),
+          List(SchemeEvaluationResult(SchemeId("Project Delivery"), EvaluationResults.Green.toPassmark)), "", Some(""))))
+    }
+
+    """return no results when there are only phase 3 applications that aren't in Passed_Notified which apply for sift or don't have Green/Passed
```

Comment:
I think if you let intellij do its standard new line stuff it'll close your quotes on the line above and open them again on the line below - having to add stripMargin smells a bit